src/Common.hpp

```diff
@@ -32,11 +36,19 @@ struct Bustools_opt {
   bool count_gene_multimapping = false;
 
   std::string capture;
+  char type;
```

Comment:
If these parameters are only used in a specific subcommand it's better to preface them with that command, since the name can be generic, so `type` should be `capture_type`. Same for other parameters like start and end below.


```suggestion
CAPTURE_TYPE capture_type;
```